app/models/grda_warehouse/data_source.rb

```diff
@@ -497,36 +497,40 @@ def unprocessed_enrollment_count
     @unprocessed_enrollment_count ||= enrollments.unprocessed.joins(:project, :destination_client).count
   end
 
-  # Have we received the expected number of files at least once over the past 48 hours?
-  # there is an assumption that within
-  # return the date of the most-recent fully successful import
-  def stalled_since?(date)
-    return nil unless date.present?
+  # Returns the date of the most recent fully successful import if the import is stalled, nil if it is not stalled
+  # @return [Date, nil] The date the import stalled, or nil if not stalled.
+  def stalled?
```

Comment:
Suggest removing the '?' since it's not returning a boolean
 ```suggestion
   def stall_date
```